analysis/src/analysis.js

```diff
@@ -46,9 +47,11 @@ class Analysis {
       this.bower.prune().then(() => {
         return this.bower.install(attributes.owner, attributes.repo, versionOrSha);
       }).then(mainHtmlPaths => {
+        const root = path.resolve(process.cwd(), 'bower_components', attributes.repo);
```

Comment:
Yeah... I wish I could be more helpful here, but I'm not 100% sure that the attributes.repo assumption is correct and I'm also not 100% sure that it's not. My memory on this is fuzzy at best.

Is there a way to verify the correctness with an assert here and then just run a full analysis on staging?